### ISSUE TITLE
Ombi cannot send requests to Radarr/Sonarr

### ISSUE THREAD

**Issue by matloke** [tidusjar/Ombi#3038]
**Describe the bug**
Using Unraid, and the Binhex Sonarr/Radarr. Installed the latest Ombi container. I have set up Plex, Sonarr and Radarr in Ombi. Plex appears to be working fine (shows items that are in my library correctly), however whenever I request a movie/tv show, the request does not get sent. I get the below logs:

Note: not using a reverse proxy or anything at this stage, no base URLs, all docker containers in bridge mode.

**Logs (Logs directory where Ombi is located)**
`at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIIndexMiddleware.Invoke(HttpContext httpContext)
at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.InvokeCore(HttpContext context)
at Ombi.ApiKeyMiddlewear.Invoke(HttpContext context) in C:\projects\requestplex\src\Ombi\ApiKeyMiddlewear.cs:line 51
at Ombi.ErrorHandlingMiddleware.Invoke(HttpContext context) in C:\projects\requestplex\src\Ombi\ErrorHandlingMiddlewear.cs:line 24`

**Comment by tidusjar** [comment]
Are there any more logs, because what you have posted above seems to be half of the stacktrace?

**Comment by matloke** [comment]
Thanks for the fast response mate! I've tried again and its shown a few more lines. Not sure if its helpful!

`at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIIndexMiddleware.Invoke(HttpContext httpContext)
at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext)
at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.InvokeCore(HttpContext context)
at Ombi.ApiKeyMiddlewear.Invoke(HttpContext context) in C:\projects\requestplex\src\Ombi\ApiKeyMiddlewear.cs:line 51
at Ombi.ErrorHandlingMiddleware.Invoke(HttpContext context) in C:\projects\requestplex\src\Ombi\ErrorHandlingMiddlewear.cs:line 24
[41m[30mfail[39m[22m[49m: Ombi.ErrorHandlingMiddleware[0]
Something bad happened, ErrorMiddleware caught this
System.NullReferenceException: Object reference not set to an instance of an object.
at Ombi.Core.Engine.MovieRequestEngine.RequestMovie(MovieRequestViewModel model) in C:\projects\requestplex\src\Ombi.Core\Engine\MovieRequestEngine.cs:line 70
at Ombi.Controllers.RequestController.RequestMovie(MovieRequestViewModel movie) in C:\projects\requestplex\src\Ombi\Controllers\RequestController.cs:line 89
at lambda_method(Closure , Object )
at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)`

**Comment by tidusjar** [comment]
Can you log out and log back in and try again?

Does this happen on every movie?

**Comment by matloke** [comment]
I've got it working. Turns out it was a user issue - I've recently changed from a Windows to an Unraid machine and in my numerous attempts of installing and reinstalling Ombi somehow I ended up logged into an instance of Ombi where no admin existed... Reinstalled container and skipped plex set-up and just made the admin account and set everything up from there. Now to reverse proxy everything again as I did on windows! 

Thanks for the attention and creating this great application! I'll send a donation tomorrow on payday :)
